zilliqa/src/crypto.rs

```diff
@@ -219,11 +219,11 @@ impl SecretKey {
         Self::from_bytes(&bytes_vec)
     }
 
-    fn as_bls(&self) -> bls_signatures::PrivateKey {
+    pub fn as_bls(&self) -> bls_signatures::PrivateKey {
```

Comment:
Minor internal interface change - but I feel like if we have access to a `SecretKey` field in some scope, there's no real reason not to have these conversion methods public for it.